File: DAM 25-26/Acceso_a_datos/SwapShop/src/main/java/com/swapshop/dto/UsuarioDTO.java
package com.swapshop.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UsuarioDTO {

    private Integer id;
    private String nombre;
    private String apellido1;
    private String apellido2;
    private String correo;
}

File: DAM 25-26/DesarolloDeInterfaces/Front/SwapShop/src/app/modelos/UsuarioDTO.ts
export interface UsuarioDTO {
  id?: number;
  nombreUsuario: string;
  nombre: string;
  apellido1: string;
  apellido2?: string;        // Opcional
  correo: string;
  contrasena?: string;      // Solo en registro/login, no se muestra después
  fecha_nac: string;        // Usaremos string en formato 'YYYY-MM-DD' para ion-datetime
}
